protocol/messenger_communities.go

```diff
@@ -4997,12 +4990,13 @@ func (m *Messenger) rekeyAllCommunities(logger *zap.Logger) {
 	// Determine the rekey interval, if the value is not set as a property of m.communitiesManager
 	// default to one hour
 	// TODO in future have a community level rki rather than a global rki
-	var rki time.Duration
-	if m.communitiesManager.RekeyInterval == 0 {
-		rki = time.Hour
-	} else {
-		rki = m.communitiesManager.RekeyInterval
-	}
+	/*
+		var rki time.Duration
+		if m.communitiesManager.RekeyInterval == 0 {
+			rki = time.Hour
+		} else {
+			rki = m.communitiesManager.RekeyInterval
+		}*/
```

Comment:
Still needed?

Reply:
not sure :) it wasn't used at all, so I left it as it was (the only usage was a printf), I would leave it for now just because I didn't quite look at it